src/base/SExpr.zig

```diff
@@ -0,0 +1,213 @@
+//! A simple format for representing tree-like data.
```

Comment:
I'd feel more comfortable here if this was an actual move.

Reply:
Yeah, it's a total re-write. I went back to first principles and redesigned it based on my experience using it. I replaced the `union(enum)` with a simple `struct` and added attributes which I think makes the formatting much nicer.